extension/data.ts

```diff
@@ -86,6 +86,27 @@ interface DeinflectionRuleGroup {
   rules: DeinflectionRule[];
 }
 
+const WORD_DICT_ENTRY_REGEX = /^(.+?)\s+(?:\[(.*?)\])?\s*\/(.+)\//;
+
+type WordDictEntry = {
+  word: string;
+  pronunciation: string | null;
+  definitions: string[];
+};
+
+export const parseWordDictEntry = (line: string): WordDictEntry | null => {
+  const [, word, pronunciation, definitions] =
+    line.match(WORD_DICT_ENTRY_REGEX) || [];
+  if (!word || !definitions) {
```

Comment:
Is there any reason we would expect this to happen normally? It seems like it can only happen in an error case or new type of dictionary entry. maybe logging a warning would be good if it's unexpected...

Reply:
It only happens if there's an improperly formatted dictionary line. Right now there's only a single line in the dictionary that's improperly formatted: `４° [しど] /`.

Yeah logging makes sense.

If we use this parsing function in update-db.ts in the future then these logs will show up and so we'll know if there are any additional improperly dictionary lines that get added when the dictionary changes in the future.